packages/dashboard/src/app/views/shared/bodyViewOptions.js

```diff
@@ -91,6 +98,30 @@ export default function BodyViewOptions({
           <TranslateWithMarkup>{resultsLabel}</TranslateWithMarkup>
         </Text>
         <ControlsContainer>
+          {layoutStyle === VIEW_STYLE.GRID && filters?.length
```

Comment:
Sorta related: wondering why we don't currently support filtering on list view. Maybe worth adding a ticket?

Reply:
For sure!